plugins/services/src/js/utils/DeclinedOffersUtil.js

```diff
@@ -201,14 +201,14 @@ const DeclinedOffersUtil = {
         unmatchedResource: declinedOffer.reason,
         offered: resources.reduce(
           (accumulator, resource) => {
-            const { name, role } = resource;
+            const { name, role, scalar = 0 } = resource;
```

Comment:
Is it possible to get `null` in `resource.scalar`? Because in this case the default value won't be assigned.